### PR TITLE
bug-1878000: reduce spam from `make run`

### PR DESCRIPTION
**Pull request by willkg**
This reduces the spam in the output when running `make run` by explicitly not attaching to backing services and improving Kent logging.

While doing that, I saw this output when running `make build`:

```
/usr/bin/docker compose build --progress plain localstack statsd
--progress is a global compose flag, better use `docker compose --progress xx build ...touch .docker-build
```

So I fixed that, too.

### CODE REVIEW

**Comment by willkg** [comment]
We should land this after PR #965 because that picks up circleci fixes including updating to a more recent version of Docker.

File: docker/images/fakesentry/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+FROM python:3.10.9-slim@sha256:6862d8ed663a47f649ba5aababed01e44741a032e80d5800db619f5113f65434
+
+ARG groupid=5000
+ARG userid=5000
```
Comment (smarnach) [inline docker/images/fakesentry/Dockerfile:4]: I think these arguments are unused now and can be removed.

**Review by smarnach — APPROVED** [review]
Looks good to me. Just wondering – is the Kent upgrade related to that change?

**Comment by willkg** [comment]
The Kent upgrade picks up changes to logging that reduce logging spam. It's not as applicable here because Antenna doesn't have the service healthchecks that Tecken has, but when we implement those, we'll want the fixes in Kent 1.2.0 that quell the logging that triggered with the healthchecks every second.

File: Makefile [hunk 2]
```diff
@@ -60,7 +59,10 @@ setup: my.env .docker-build  ## | Set up services.
 
 .PHONY: run
 run: my.env .docker-build  ## | Run the webapp and services.
-	${DC} up web fakesentry
+	${DC} up \
+		--no-attach statsd \
+		--no-attach localstack \
+		web fakesentry
```
[the remark this answers is not in the document]
Comment (willkg) [reply Makefile:65]: We worked this out in https://github.com/mozilla-services/tecken/pull/2890 . I'm going to change this PR to do what we did there.

**Review by relud — APPROVED** [review]

**Comment by willkg** [comment]
Thank you!

**Pull request merged**